tests/mobly/controllers/android_device_lib/adb_test.py

```diff
@@ -285,7 +285,7 @@ def test_execute_and_process_stdout_when_handler_crash(self, mock_popen):
     mock_handler.side_effect = ['', TypeError('fake crash'), '', '']
 
     with self.assertRaisesRegex(TypeError, 'fake crash'):
-      err = adb.AdbProxy()._execute_and_process_stdout(['fake_cmd'],
+      _ = adb.AdbProxy()._execute_and_process_stdout(['fake_cmd'],
```

Comment:
why keep the assignment at all?
same for elsewhere

Reply:
This is to inform readers these return values are intentionally dropped, rather than erronously forgotten about.

Reply:
I don't think it's needed since we are not doing anything with the return values.
Could you remove all?